CCP_App/src/main/java/com/example/demo2/viewmodel/UserViewModel.java

```diff
@@ -0,0 +1,49 @@
+/************************************************************************
+ *                                                                      *
+ *  DDDD     SSSS    AAA        Daten- und Systemtechnik Aachen GmbH    *
+ *  D   D   SS      A   A       Pascalstrasse 28                        *
+ *  D   D    SSS    AAAAA       52076 Aachen-Oberforstbach, Germany     *
+ *  D   D      SS   A   A       Telefon: +49 (0)2408 / 9492-0           *
+ *  DDDD    SSSS    A   A       Telefax: +49 (0)2408 / 9492-92          *
+ *                                                                      *
+ *                                                                      *
+ *  (c) Copyright by DSA - all rights reserved                          *
+ *                                                                      *
+ ************************************************************************
+ *
+ * Author : Riyaardh Adam
+ * Created: 2023/02/21 12:26
+ *
+ ***********************************************************************/
+package com.example.demo2.viewmodel;
+
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.Setter;
+
+import javax.validation.constraints.Email;
+import javax.validation.constraints.Max;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+
+@Getter
+@Setter
+public class UserViewModel {
+	private String id;
+
+	@NotNull
+	@NotBlank
+	private String firstName;
+	@NotNull
+	@NotBlank
+	private String lastName;
+
+	@NotNull
+	@NotBlank
+	private String userName;
+	@Email(regexp = ".+[@].+[\\.].+")
```

Comment:
If your regex is not a more specific implementation of thet @Email implementation I would sugggest leaving it default so the standard pattern matching is applied